### DISCUSSION TITLE
Find products with no tags in the tag array?

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#169] [Help] [answered]
I tried this:

`GET /products`

```
{
  "where": {
    "tags": {
      "$size": 0
    }
  }
}
```

but to fish.

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi @haf, your query looks correct, you can also use a short-handed version, like:

```json
{
  "tags": { "$size": 0 }
}
```

It should work on both Frontend and Backend APIs.
